name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/JustTradePDFExtractor.java

```diff
@@ -15,7 +18,7 @@
 
 public class JustTradePDFExtractor extends AbstractPDFExtractor
 {
-
+    private static final String FLAG_WITHHOLDING_TAX_FOUND  = "false"; //$NON-NLS-1$

```

Comment:
Das ist der Schlüssel in die Map - und der ist damit ```false```.
Es macht auf jeden Fall Sinn eine Konstante zu definieren, aber eher identisch zu dem Namen der Konstante.

Reply:
:sweat_smile: